documentation/release_notes.markdown

```diff
@@ -37,6 +38,15 @@ canonical: "/puppetdb/latest/release_notes.html"
   expected, and is due to internal changes in the way we check if
   catalogs are up-to-date.
 
+### Downgrading
+
+* If you attempt to downgrade to a previous version of PuppetDB, in
+  addition to wiping the database, you will need to clear the queue
+  directory as described in the
+  [documentation on KahaDB corruption][kahadb_corruption].  If you see
```

Comment:
might be good to have a sentence explaining what change occurred to prompt this warning.